src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecutionPlanGraphUtils.cs

```diff
@@ -100,6 +109,54 @@ public static ExecutionPlanEdges ConvertShowPlanEdgeToExecutionPlanEdge(Edge edg
             };
         }
 
+        private static string GetEstimatedRowsForAllExecutions(PropertyValue? property)
+        {
+            var estimatedRowsForAllExecutions = string.Empty;
+
+            if (property != null)
+            {
+                estimatedRowsForAllExecutions = GetPropertyDisplayValue(property);
+            }
+
+            return estimatedRowsForAllExecutions;
+        }
+
+        private static string GetEstimatedRowsRead(PropertyValue? property)
+        {
+            var estimatedRowsRead = string.Empty;
+
+            if (property != null)
+            {
+                estimatedRowsRead = GetPropertyDisplayValue(property);
+            }
+
+            return estimatedRowsRead;
+        }
+
+        private static string GetActualRows(PropertyValue? property)
+        {
+            var actualRows = string.Empty;
+
+            if (property != null)
+            {
+                actualRows = GetPropertyDisplayValue(property);
+            }
+
+            return actualRows;
+        }
+
+        private static string GetActualRowsRead(PropertyValue? property)
```

Comment:
These all appear to be doing exactly the same thing - I'd just rename this as `GetPropertyDisplayValue` and only have a single one.